manifests/m/martinchrzan/ColorPicker/1.1.1.0/martinchrzan.ColorPicker.locale.en-US.yaml

```
@@ -0,0 +1,27 @@
+# Created using YamlCreate.ps1 v1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: martinchrzan.ColorPicker
+PackageVersion: 1.1.1.0
+PackageLocale: en-US
+Publisher: Martin Chrzan
+PublisherUrl: https://github.com/martinchrzan
+PublisherSupportUrl: https://github.com/martinchrzan/ColorPicker/issues
+#PrivacyUrl: 
+Author: martinchrzan
+PackageName: ColorPicker
```

Looks correct other than this
```suggestion
PackageName: Color Picker
```